include/avif/internal.h

```diff
@@ -42,10 +42,16 @@ extern "C" {
 
 float avifRoundf(float v);
 
+// H is platform-dependent. Could be little- or big-endian.
+// N is big-endian: most- to least-significant bytes.
+// Z is little-endian: most- to least-significant bytes.
```

Comment:
"H" and "N" come from network programming. "H" stands for "host", i.e., the computer. "N" stands for "network". The network byte order is big endian by definition. The `avifHTONS`, `avifNTOHS`, ... function names come from the Berkeley sockets API, which has the `htons`, `ntohs`, ... functions.

I have not heard of "Z". What does it stand for?

Is the comment for "Z" (little-endian) incorrect? It is the same as the comment for big-endian: "most- to least-significant bytes".

Reply:
> "H" and "N" come from network programming. "H" stands for "host", i.e., the computer. "N" stands for "network". The network byte order is big endian by definition. The `avifHTONS`, `avifNTOHS`, ... function names come from the Berkeley sockets API, which has the `htons`, `ntohs`, ... functions.

Thanks for the context. I searched for that information but failed to obtain it on my own.

> I have not heard of "Z". What does it stand for?

I had no clue what H/N were so took another random letter. Let's take "C" for "custom".

> Is the comment for "Z" (little-endian) incorrect? It is the same as the comment for big-endian: "most- to least-significant bytes".

Fixed.

Reply:
Here is the man page for the `htons`, `ntohs`, ... functions:

https://pubs.opengroup.org/onlinepubs/9699919799/functions/htonl.html

Their names are cryptic for people unfamiliar with socket programming. We can switch to more descriptive names if we are adding little-endian variants of these functions. But let's focus on correctness of this pull request first and deal with these function names later.